src/main/java/mate/jdbc/service/DriverServiceImpl.java

```diff
@@ -18,7 +18,8 @@ public Driver create(Driver driver) {
 
     @Override
     public Driver get(Long id) {
-        return driverDao.get(id).get();
+        return driverDao.get(id).orElseThrow(() ->
+                new RuntimeException("Useless method get."));
```

Comment:
Let's change the message to "Couldn't get a driver from DAO by id = " + id

Reply:
Added